index.html

```diff
@@ -2,253 +2,291 @@
-                                        HTML
-                                    </div>
-                                </div>
-                                <div class="item">
-                                    <div class="logo">
-                                        <img src="./assets/img/css.svg" alt="css" />
-                                    </div>
-                                    <div>
-                                        CSS
-                                    </div>
-                                </div>
-                            </div>
-                        </li>
-                        <li class="categories">
-                            <a href="#">
-                                <span>
-                                    Frameworks
-                                </span>
-                                <span><img src="./assets/img/arrow-right.png" alt="more" /></span>
-                            </a>
-                        </li>
-                        <li class="categories">
-                            <a href="#"><span>Skills</span><span><img src="./assets/img/arrow-right.png"
-                                        alt="more" /></span></a>
-                        </li>
-                    </ul>
-                </div>
-            </section>
-        </main>
-    </div>
+              </div>
+            </li>
+            <li class="categories">
+              <a href="#">
+                <span> Frameworks </span>
+                <span><img src="./assets/img/arrow-right.png" alt="more" /></span>
+              </a>
+            </li>
+            <li class="categories">
+              <a href="#"><span>Skills</span><span><img src="./assets/img/arrow-right.png" alt="more" /></span></a>
+            </li>
+          </ul>
+        </div>
+      </section>
+      <section id="contact-me">
+        <div class="contact-me">
+          <div class="wrapper">
+            <div>
+              <h3>Contact me</h3>
+              <p>
+                If you have an application you are interested in developing, a
+                feature that you need built or a project that needs coding. I’d love
+                to help with it
+              </p>
+              <form action="https://formspree.io/f/moqzvdbe" method="POST">
+                <label for="name">
+                  <input type="text" id="name" placeholder="Full name" maxlength="30" minlength="3" required />
+                </label>
+                <label for="email">
+                  <input type="email" id="email" placeholder="Enter the email" required />
+                </label>
+                <label for="message">
+                  <textarea id="message" cols="30" rows="7" maxlength="500" minlength="5" required
+                    placeholder="Write your message here"></textarea>
+                </label>
+                <button type="submit">Get in touch</button>
+              </form>
```

Comment:
- [ ] Amazing work with the design. Currently, the form is not getting submitted because the inputs don't have a `name` attribute. For the Formspree service to work correctly, we are supposed to add `name` attribute on our form elements. 🧐️
Kindly add `name` attribute on all inputs and the textarea to fix the issue.

**Current behaviour**
<img src="https://user-images.githubusercontent.com/17243325/213184427-de79ab92-ebcb-40ed-97a5-5b91da355fe7.png" width="300">

Reply:
Thanks for the feedback. I have added the name attributes to the fields now